### ISSUE TITLE
Examples fail when checking package

### ISSUE THREAD

**Issue by markkvdb** [markkvdb/ghoR#15]
```
 Running examples in ‘ghoR-Ex.R’ failed
  The error most likely occurred in:
  
  > base::assign(".ptime", proc.time(), pos = "CheckExEnv")
  > ### Name: download_ind
  > ### Title: Download file of indicator and read it into a data frame
  > ### Aliases: download_ind
  > 
  > ### ** Examples
  > 
  > download_ind("RSUD_30")
  Error in download_ind("RSUD_30") : could not find function "download_ind"
  Execution halted
```

**Comment by markkvdb** [comment]
Might have something to do with the fact that we don't export these functions

**Comment by markkvdb** [comment]
fixed in 16cb9755a01dfeb22c9fb831270dc34cce968964
